app/api/sessions.py

```diff
@@ -269,7 +270,10 @@ def before_update_object(self, session, data, view_kwargs):
 
         new_state = data.get('state')
 
-        if new_state and new_state != session.state:
+        if new_state and new_state != session.state and (
+            is_organizer or (
+            session.ends_at and 
+            session.ends_at.date() > datetime.now().date())):
```

Comment:
Still wrong. Read the error carefully

Reply:
test is fixed. And its working correctly too

Reply:
session.ends_at > datetime.utcnow()